### ISSUE TITLE
The button icon in the new chat button is not hidden.

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3819]
**Issue Description**:
The button icons in the new chat button are not hidden, causing assistive technology to read the icon as content. Since the button is already named by its aria-label attribute, the icon does not provide important information and should be hidden. 
Related to issue #3817 

**WCAG Criteria**:
[1.1.1 Non-text Content - A](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content)

**Screenshot or Screen Recording**:

<img width="1680" alt="Button Icon" src="https://github.com/user-attachments/assets/c5a0bc2e-2c3d-4846-a579-0d0c3cc735ac">

`
<button id="nav-new-chat-btn" aria-label="nav-new-chat-btn" class="text-token-text-primary" data-state="closed">
<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" class="text-black dark:text-white size-5">`

Notes:
Add the aria-hidden="true" attribute to the svg tag. Icons in these buttons should be hidden to prevent them from being announced by screen readers unless they convey important information, such as the button's name.
